app/views/taxons/show_c.html.erb

```diff
@@ -1,5 +1,69 @@
-<%= render template: "taxons/show_a",
-  locals: {
-    presented_taxon: presented_taxon
-  }
+<% content_for :page_class, "taxon-page taxon-page--grid" %>
+<% content_for :is_taxon_with_subtopics, presented_taxon.show_subtopic_grid? %>
+<%=
+  render(
+      partial: 'common',
+      locals: {
+          presented_taxon: presented_taxon
+      }
+  )
 %>
+
+<div class="full-page-width-wrapper">
+  <div class="grid-row">
+    <div class="column-one-third">
+      <%= render "govuk_publishing_components/components/contents_list", {
+          hide_title: true,
+          contents: [
+              {
+                  text: "On this page:",
+                  active: true,
+                  items: @presentable_section_items
+              }
+          ]
+      } %>
+    </div>
+    <div class="column-two-thirds">
+      <% presented_taxon.sections.each do |section| %>
+        <% if section[:show_section] %>
+          <div class="taxon-page__section-group">
+            <div class="grid-row">
+              <div class="column-two-thirds">
+                <%= render "govuk_publishing_components/components/heading", {
+                    text: t(section[:id], scope: :content_purpose_supergroup, default: section[:title]),
+                    heading_level: 2,
+                    margin_bottom: 2,
+                    id: section[:id]
+                } %>
+              </div>
+            </div>
+            <%= render(
+                    partial: section[:partial_template],
+                    locals: {section: section}
```

Comment:
We normally try and keep a space between the `{` and the content, e.g: `{ section: section }`.

Reply:
👍